environment.json.schema

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://dl-platform-public-schemas/v0.1.0/environment.json.schema",
+    "title": "Notebook Environment Representation Schema",
+    "description": "This standard represents information about the environment that is required for the Notebook to be reproducible.",
+    "type": "object",
+    "items": {
+        "python_packages": {
+            "type": "array",
+            "items": {
+                "type": "string"
```

Comment:
What if we make this a bit more specific to specify both package name and package version (to ensure it always points to the same version):

I believe the schema for that section would look something like:

       "items": {
         "type": "object",
         "title": "Python packages with exact versions",
         "required": [
           "name",
           "version"
         ],
         "properties": {
           "name": {
             "type": "string",
             "title": "package version",
           },
           "version": {
             "type": "string",
             "title": "exact package version"
           }
         }